Automattic/jetpack
# Calypsoify: Remove the user meta setting

#### Changes proposed in this Pull Request:

As a first step towards removing Calypsoify, we are removing the
mechanisms by which the feature is persisted between requests. In
Jetpack this is done by setting a user meta value.

This PR removes that and deletes the user meta if it's present. After
this PR is merged Calypsoify is only activated when the `calypsoify`
query parameter is present and if we're on a Gutenberg screen `post-new.php` etc.

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

The main way to tell if Calypsoify is activated is via the close URL on the Gutenberg editor. 

* Without this PR, and an account/site without nav unification, load Gutenberg via Calypso so the `calypsoify=1` parameter will be set.
* Go to a page in `wp-admin` but not `wp-admin/` itself, e.g. `wp-admin/plugins.php`
* You should see that you're in a Calypsoified state
* Apply this PR, and reload the page, it should be reset back to the standard admin
* Open the editor and click the WordPress logo to close it. This should take you back to the `wp-admin` posts list.
* Open the editor from within Calypso and click the WordPress logo to close it. You should be taken back to the Calypso posts list.

#### Proposed changelog entry for your changes:

* Removed the Calypsoify setting mechanism.
